_includes/tutorials/generate-test-data-streams/ksql/code/src/statements.sql

```diff
@@ -1,26 +1,38 @@
-CREATE SOURCE CONNECTOR IF NOT EXISTS CLICKS WITH (
-    'connector.class'             = 'io.mdrogalis.voluble.VolubleSourceConnector',
-    'key.converter'               = 'org.apache.kafka.connect.storage.StringConverter',
-    'genkp.clicks.with'           = '#{Number.randomDigit}',
-    'attrkp.clicks.null.rate'     = 1,
-    'genv.clicks.source_ip.with'  = '#{Internet.ipV4Address}',
-    'genv.clicks.host.with'       = '#{Internet.url}',
-    'genv.clicks.path.with'       = '#{File.fileName}',
-    'genv.clicks.user_agent.with' = '#{Internet.userAgentAny}',
-    'topic.clicks.throttle.ms'    = 1000 
+CREATE SOURCE CONNECTOR IF NOT EXISTS PIZZA_ORDERS WITH (
```

Comment:
suggest aligning all the `=`'s for readability here and in the two statements below